public/css/widget/donut-container.less

```diff
@@ -12,4 +14,8 @@
   &:not(:last-of-type) {
     margin-right: 1em;
   }
+
+  .donut {
+    margin: 0 auto;
```

Comment:
What effect has this? I can't see one.

Reply:
Makes sense, this was missing:
https://github.com/Icinga/icingadb-web/blob/0c2b7b9e74fadea3a96c9f76fd8e14b8802a4cf2/public/css/widget/donut-container.less#L22

See the difference:
![Desktop Copy 4](https://user-images.githubusercontent.com/6977434/140297191-f27ab32d-53c8-424f-aca7-e87e7a84a840.jpg)

Reply:
✅